src/Glpi/Form/ServiceCatalog/Provider/CompositeProviderInterface.php

```diff
@@ -35,7 +35,15 @@
 
 namespace Glpi\Form\ServiceCatalog\Provider;
 
+use Glpi\Form\ServiceCatalog\ItemRequest;
+use Override;
+
 /**
  * @template T of \Glpi\Form\ServiceCatalog\ServiceCatalogCompositeInterface
```

Comment:
Maybe this would work.

```suggestion
 * @implements ItemProviderInterface<\Glpi\Form\ServiceCatalog\ServiceCatalogCompositeInterface>
```